common/honeycomb/dummy_honeycomb.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package honeycomb
+
+type BatchPointsSavedToHoneycomb struct {
+	BatchPoint *BatchPoint
+}
+
+type FakeHoneycombClient struct {
+	BatchPoints []BatchPointsSavedToHoneycomb
+}
+
+func NewFakeHoneycombClient() *FakeHoneycombClient {
+	return &FakeHoneycombClient{[]BatchPointsSavedToHoneycomb{}}
+}
+
+func (client *FakeHoneycombClient) SendBatch(batch Batch) error {
+	for _, batchpoint := range batch {
+		client.BatchPoints = append(client.BatchPoints, BatchPointsSavedToHoneycomb{batchpoint})
+	}
+	return nil
+}
+
+var FakeClient = NewFakeHoneycombClient()
```

Comment:
Hmm... package-level globals are generally suspect. It'd be better to call `NewFakeHoneycombClient()` where this object is used, in the `NewFakeClient()` functions. Otherwise, someone may add more tests down the road and be confused as to why state carries over between tests.